aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -112,14 +81,34 @@ RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema) {
 
   OperatorName op_name = schema.operator_name();
 
-  auto op = findOrRegisterSchema_(std::move(schema));
+  const auto found = findSchema(op_name);
```

Comment:
Maybe rename this to `findSchemaOrName` since it will also return operators that had an impl() but not a def() yet. Or maybe `findOperatorForName` or something like that?

Reply:
findOperatorForName seems better.

Reply:
This rename would be pretty invasive, as `findSchema` here is the literal top level findSchema that many clients use

```
  // ------------------------------------------------------------------------
  //
  // Accessing operators by schema
  //
  // ------------------------------------------------------------------------

  /**
   * Looks for an operator schema with the given name and overload name
   * and returns it if it is registered.
   * Returns nullopt otherwise.
   */
  c10::optional<OperatorHandle> findSchema(const OperatorName& operator_name);
```

I'm going to file an issue to do it later. https://github.com/pytorch/pytorch/issues/35038

Reply:
What I'm actually going to do, is findSchema will only return if there is actually a schema, and findOperatorForName will give you a thing always

Reply:
I think it generally makes sense to also rename the top level one, `findSchema` is not accurate anymore.